src/Halogen/Menu/Component.purs

```diff
@@ -0,0 +1,124 @@
+module Halogen.Menu.Component where
```

Comment:
Yeah, so like `CellState` or whatever. However, the other option is to just use `State` as the name and import qualified if it needs disambiguation. That's how Maxim did things in the FS component and that works well too.